tools/stripeSetup.js

```diff
@@ -0,0 +1,22 @@
+require('dotenv/config');
```

Comment:
This should be moved to server module.

Reply:
@mitjade Are you sure? I think `tools/stripeSetup.js` is used from `package.json` script `stripe:setup`

Reply:
@vlasenko Can't we change the path there also?

Reply:
@mitjade Sure we can, but should we put this code into server module, is it right? This is technically not a part of backend, but rather a helper script

Reply:
@vlasenko I guess we need to define a way how to handle this kind of cases, since it is something other complex modules will likely require as well.

Reply:
My thoughts were the same. I wasn't sure where it needed to live, `tools` seemed better than client or server. Although, that does kind of break the modular concept. I'm happy to move it wherever you decide.